README.md

```diff
@@ -1,2 +1,267 @@
-# Microverse_code-review
-Learning how to review code, a project from the Microverse curriculum
+<a name="readme-top"></a>
+
+
+
+
+
+<!-- TABLE OF CONTENTS -->
```

Comment:
Please remove unnecessary comments in your codebase. This will allow you to have a clean code.